tests/nightly/model_backwards_compatibility_check/train_mxnet_legacy_models.sh

```diff
@@ -25,14 +25,14 @@ run_models() {
 	echo '=========================='
 	echo "Running training files and preparing models"
 	echo '=========================='
-	python model_backwards_compat_train.py
+	python3 model_backwards_compat_train.py
 	echo '=========================='
 }
 
 install_mxnet() {
 	version=$1
 	echo "Installing MXNet "$version
-	pip install mxnet==$version --user
+	pip3 install mxnet==$version --user
```

Comment:
Does this resolve the Python module not found issues we are seeing in the MBCC Train Stage on Jenkins ?